.circleci/config.yml

```diff
@@ -12,3 +12,29 @@ workflows:
             # Trigger job also on git tag.
             tags:
               only: /^v.*/
+
+      - architect/push-to-app-catalog:
+          context: "architect"
+          name: push-apptest-app-to-control-plane-catalog
+          app_catalog: "control-plane-test-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "apptest-app"
+          filters:
+            # Do not trigger the job on merge to master.
+            branches:
+              ignore:
+                - master
```

Comment:
@tomahawk28 I updated the test to only push to the test catalog and skip merges to master. As the chart is only used for the test.

PTAL